RACK-Ontology/OwlModels/SACM-S.owl

```diff
@@ -18,12 +18,12 @@
     <rdfs:comment xml:lang="en">This ontology was created from a SADL file 'SACM-S.sadl' and should not be directly edited.</rdfs:comment>
   </owl:Ontology>
   <owl:Class rdf:ID="ASSERTED_CONTEXT">
-    <rdfs:comment xml:lang="en">Asserted Context can be used to declare that the artifact cited by an ArtifactReference(s) provides the context for the
+    <rdfs:comment xml:lang="en">Asserted Context can be used to declare that the artifact cited by an ArtifactReference(s) provides the context for the&#xD;
 interpretation and scoping of a Claim or ArgumentReasoning element.</rdfs:comment>
```

Comment:
What is this change?  It doesn't seem to correspond to a change in any .sadl file, and it doesn't seem like a change we want?